mqtt/mqtt-bridge/tests/config.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
this file is no longer needed after my changes, right?

Reply:
yeap, I added during merge and then I forgot to remove